### ISSUE TITLE
Use options defined in the .babelrc file

### ISSUE THREAD

**Issue by ryansobol** [babel/babel-brunch#41]
It would be pretty sweet if the `babel-brunch` plugin could use the options defined in the `.babelrc` file, if it exists. Something like this should work.

``` js
JSON.parse(fs.readFileSync(path.join(__dirname, '.babelrc'), 'utf8'));
```

I haven't thought much about the load order of using `.babelrc` options vs `brunch-config.js` options. I'll let smarter people who have stronger opinions worry about that.

**Comment by paulmillr** [comment]
yep

**Comment by rafaelkallis** [comment]
Dunno why this hasn't been mentioned yet, but it is possible. Issue can be closed IMO

add this in your `brunch-config.js`:

```
plugins: {
    babel: {
        babelrc: true
    }
}
```

**Comment by denysdovhan** [comment]
Could be closed. 

`babel-brunch` uses options from `.babelrc` as default.

**Comment by denysdovhan** [comment]
**UPD:** Confirmed. Related to babel/babel-brunch#52.

When we specify options only in `.babelrc`, then [these lines](https://github.com/babel/babel-brunch/blob/master/index.js#L58-L63) add `['es2016', 'es2016']` presets. Options passed to `babel.transform()` have more priority, so settings from `.babelrc` are ignored.

**Comment by denysdovhan** [comment]
Should be fixed by #53.

**Comment by denysdovhan** [comment]
#53 has been merged. This issue is resolved.
